internal/pkg/addon/addons.go

```diff
@@ -21,7 +21,7 @@ const (
 
 var (
 	wkldAddonsParameterReservedKeys = []string{"App", "Env", "Name"}
-	envAddonsParameterReservedKeys  = []string{"App", "Name"}
+	envAddonsParameterReservedKeys  = []string{"App", "Env"}
```

Comment:
Given that people have been writing workload addons, in which they have been referring to env via `${Env}` in their workload addons, I was assuming they'd more naturally written `${Env}`  instead of `${Name}` to refer to the name of an env in the env addons as well.

The example that I was mentally running through was when users want to export some `Outputs`, they'd probably write:
```yaml
Outputs:
  DBSecurityGroup:
     Value: !DBSecurityGroup
     Name: !Ref ${App}-${Env}-SecurityGroup # vs  Name: !Ref ${App}-${Name}-SecurityGroup
```

The `${Env}` version looks clearer that it's namespaced by env name.